todo_app/templates/confirmation.html

```diff
@@ -0,0 +1,16 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+
+{% block content %}
+  <div class="jumbotron">
+    <h1 class="display-4">To-Do App</h1>
+    <p class="lead">Just another to-do app.</p>
+  </div>
+
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Item added to your to-do list.</h2>
+      <p><a href="http://localhost:5000">Go back</a></p>
```

Comment:
This href should be relative, so that it works on in any environment. i.e. `href="/"`

Or you can use `href="{{url_for('index')}}" to specify it by function name

Reply:
removed localhost